src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -72,7 +72,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       RestoreDisableParallel="$(RestoreDisableParallel)"
       RestoreConfigFile="$(RestoreConfigFile)"
       RestoreNoCache="$(RestoreNoCache)"
-      RestoreIgnoreFailedSources="$(RestoreIgnoreFailedSources)" />
+      RestoreIgnoreFailedSources="$(RestoreIgnoreFailedSources)"
+      RestoreNoRecurse="$(RestoreNoRecurse)" />
```

Comment:
Yeah, couldn't this just be `$(RestoreRecurse)`? and then negate later if necessary